AmericanMedicalAssociation/AMA-style-guide
# [EWL-4133] Podcast | Design Podcast Promotion Component

## Ticket(s)


**Jira Ticket**

- [EWL-4133: Podcast | Design Podcast Promotion Component](https://issues.ama-assn.org/browse/EWL-4133)


## Description

Adds a new molecule, Hub Card with Select, and includes it in the Hub Card Row pattern.


## To Test
- [ ] Make sure the select menu @atom ( http://localhost:3000/?p=atoms-select-menu )looks the same as --> https://americanmedicalassociation.github.io/AMA-style-guide/?p=atoms-select-menu .

- [ ] Molecules > Hub Components > Hub Card with Select: verify that the pattern exists and that it displays appropriately according to the attachments in the JIRA ticket 
- [ ] Organisms > Hub > Hub Card Row: verify that the Hub Card with Select pattern is included and displays appropriately in the organism
- [ ] Templates > Hub > Hub: scroll down and verify that the Hub Card with Select pattern displays appropriately 
- [ ] Templates > Hub > Hub page: scroll down and verify that the Hub Card with Select pattern displays appropriately ( http://localhost:3000/patterns/03-templates-03-hub-hub-page/03-templates-03-hub-hub-page.html?1510084455756 )

- [ ]  - Verify that when options are selected on http://localhost:3000/patterns/03-templates-03-hub-hub-page/03-templates-03-hub-hub-page.html?1510084455756 that they redirect to the specified 'option'.

- Select an option
- iTunes|https://www.apple.com/itunes/download/
- Google Play|https://play.google.com/store?hl=en
- Soundcloud|https://soundcloud.com/

Should **input-select.twig** and **input-state-zip.twig** follow the same convention?


## Relevant Screenshots/GIFs
![4133-1](https://user-images.githubusercontent.com/12160398/32460140-6658327a-c2f7-11e7-953a-c96875e80a28.png)



## Remaining Tasks
- [ ] Can @froboy check what's built here and see if any behaviors or content need to be added to it? 


## Additional Notes

- I used the existing style guide conventions/structure for what was in place for the Hub. It's likely this pattern won't be migrated or will be refactored if it is.
